package.json

```diff
@@ -37,7 +39,9 @@
     "install": "node-pre-gyp install --fallback-to-build=false || make node",
     "test": "tape platform/node/test/js/**/*.test.js",
     "test-memory": "node --expose-gc platform/node/test/memory.test.js",
-    "test-suite": "node platform/node/test/render.test.js && node platform/node/test/query.test.js"
+    "test-suite": "run-s test-render \"test-render -- --recycle-map\" test-query",
```

Comment:
This will overwrite the artifacts from the first `test-render` run with those from the `test-render --recycle-map` run. Might be confusing/make it hard to debug.

Also, looks like we've roughly doubled the performance of render tests, but are giving that right back by running them twice. Should we parallelize this somehow? For instance we could run one variant in each of the two node 6 builds.

Reply:
Let's make it so then that tests results run via `--recycle-map` will be saved in `index-recycle-map.html` then.

I'm also 👍  in having e.g. node6-debug running tests with `--recycle-map` while node6-release runs them as usual.

Reply:
This is now done in GL JS side via https://github.com/mapbox/mapbox-gl-js/pull/5195. I'll update this PR accordingly.